development-defaults.ini

```diff
@@ -1,5 +1,5 @@
 
-path = "/uber"
```

Comment:
Great idea! Especially as Uber-the-car-company gets dragged through the mud due to their, um, Unique Corporate Culture (TM), it'd be nice to stop using the name.